FAIL
Kernel panic in run: PANIC at ../../threads/synch.c:258 in lock_release(): assertion `lock_held_by_current_thread (lock)' failed.
Call stack: 0xc0028ab9 0xc0023067 0xc002b336 0xc0021449
Translation of call stack:
0xc0028ab9: debug_panic (.../../lib/kernel/debug.c:38)
0xc0023067: lock_release (...../../threads/synch.c:260)
0xc002b336: load (...../userprog/process.c:348)
0xc0021449: kernel_thread (..../../threads/thread.c:570)
